src/gui/options_imp.h

```diff
@@ -164,6 +172,7 @@ private slots:
     bool webUIAuthenticationOk();
 
 private:
+    Ui::OptionsDialog *ui;
```

Comment:
I know that currently we have 2 names for this in the code `ui` and `m_ui`. Shouldn't it be named `m_ui` since it is a member variable?